webpack/scenes/AlternateContentSources/Create/Steps/AcsUrlPaths.js

```diff
@@ -12,24 +13,43 @@ import { areSubPathsValid, isValidUrl } from '../../helpers';
 
 const AcsUrlPaths = () => {
   const {
-    url, setUrl, subpaths, setSubpaths,
+    acsType, url, setUrl, subpaths, setSubpaths,
   } = useContext(ACSCreateContext);
 
   const subPathValidated = areSubPathsValid(subpaths) ? 'default' : 'error';
-  const urlValidated = isValidUrl(url) ? 'default' : 'error';
+  const [urlValidated, setUrlValidated] = React.useState('default');
+  const handleUrlChange = (newUrl, _event) => {
+    setUrl(newUrl);
+    if (isValidUrl(newUrl, acsType)) {
+      setUrlValidated('success');
+    } else {
+      setUrlValidated('error');
+    }
+  };
+
+  const baseURLplaceholder = acsType === 'rhui' ?
+    'https://rhui-server.example.com/pulp/content' :
+    'http:// or https://';
+  const helperTextInvalid = acsType === 'rhui' ?
+    'http://rhui-server.example.com/pulp/content or https://rhui-server.example.com/pulp/content' :
+    'http://, https:// or file://';
+  let headerDescription = 'Enter in the base path and any subpaths' +
+    ' that should be searched for alternate content.';
+  headerDescription = acsType === 'rhui' ?
+    `${headerDescription} The base path must be a web address pointing to the root RHUI content directory.` :
+    `${headerDescription} The base path can be a web address or a filesystem location.`;
 
   return (
     <>
       <WizardHeader
         title={__('URL and paths')}
-        description={__('Enter in the base path and any subpaths that should be searched for alternate content. ' +
-          'The base path can be a web address or a filesystem location.')}
+        description={__(headerDescription)}
```

Comment:
Same here for translations. These dynamic translations won't work. We can only translate static strings.